Examples/UIExplorer/SegmentedControlIOSExample.js

```diff
@@ -105,11 +105,11 @@ var EventSegmentedControlExample = React.createClass({
           Value: {this.state.value}
         </Text>
         <Text style={styles.text} >
-          Index: {this.state.selectedIndex}
+          Index: {this.state.selectedSegmentIndex}
         </Text>
         <SegmentedControlIOS
           values={this.state.values}
-          selectedIndex={this.state.selectedIndex}
+          selectedSegmentIndex={this.state.selectedSegmentIndex}
```

Comment:
Strangely enough, the prop name is actually `segmentedIndex`. Maybe we should change this so it is consistent on both ends.

[RCTSegmentedControl](https://github.com/facebook/react-native/blob/master/React/Views/RCTSegmentedControl.m#L52)
[SegmentedControlIOS](https://github.com/facebook/react-native/blob/master/Libraries/Components/SegmentedControlIOS/SegmentedControlIOS.ios.js#L47)